src/isIsogram.test.js

```diff
@@ -6,4 +6,28 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('Should accept empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('Should accept string with space', () => {
+    expect(isIsogram(' ')).toBe(true);
+  });
+
+  it('Should not accept string with spaces', () => {
+    expect(isIsogram('   ')).toBe(false);
+  });
+
+  it('Should not accept string with doubled letters', () => {
+    expect(isIsogram('banana')).toBe(false);
+  });
+
+  it('Should be case sensetive', () => {
```

Comment:
There's a typo in the test description: 'Should be case sensetive'. It should be 'Should be case sensitive'.